### PR TITLE
Dispose tagger when we're done with it.

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
Paging @jasonmalinowski

File: src/VisualStudio/Core/Def/Implementation/LanguageService/AbstractLanguageService`3.cs [hunk 1]
```diff
@@ -145,10 +145,12 @@ protected override void SetupNewTextView(IVsTextView textView)
                         var snapshot = subjectBuffer.CurrentSnapshot;
                         var fullSpan = new SnapshotSpan(snapshot, start: 0, length: snapshot.Length);
                         var tagger = outliningTaggerProvider.CreateTagger<IOutliningRegionTag>(subjectBuffer);
+                        using (var disposable = tagger as IDisposable)
```
Comment (jasonmalinowski) [inline src/VisualStudio/Core/Def/Implementation/LanguageService/AbstractLanguageService`3.cs]: I'd be OK with a hard cast here -- if this doesn't implement IDisposable we've got problems.

**Comment by jasonmalinowski** [comment]
:+1:

**Comment by jasonmalinowski** [comment]
Reviewers: @balajikris, @basoundr, @brettfo, @davkean, @DustinCampbell, @dpoeschl, @Pilchie

**Pull request merged**